docs/tutorials/multi-turn.md

```diff
@@ -0,0 +1,388 @@
+<!--
+SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+SPDX-License-Identifier: Apache-2.0
+-->
+
+# Multi-Turn Conversations
+
+Multi-turn conversations allow you to benchmark chat-based models with realistic back-and-forth dialogue patterns. This feature simulates real-world scenarios where users engage in extended conversations with multiple exchanges, rather than isolated single-turn queries.
+
+## Overview
+
+Multi-turn benchmarking provides several advantages:
+
+- **Realistic Chat Simulation**: Model actual user interactions with conversational AI systems
+- **Context Window Testing**: Evaluate performance as conversation history grows
+- **Session-based Load**: Test how servers handle sustained multi-turn sessions
+- **Memory and State Management**: Identify issues with conversation state handling
+- **Conversation Flow Analysis**: Measure performance degradation over multiple turns
+
+> [!IMPORTANT]
+> **Understanding Request Control Options**
+>
+> AIPerf provides different options for controlling the number of requests depending on whether you're running single-turn or multi-turn benchmarks:
+>
+> - **`--request-count`**: Controls the total number of **single-turn requests** to send. Use this for traditional single-request benchmarks.
+> - **`--conversation-num`**: Controls the total number of **conversations (sessions)** to send in multi-turn scenarios. Each conversation may contain multiple turns (requests).
+>
+> These options are mutually exclusive in their intent - use `--request-count` for single-turn benchmarking and `--conversation-num` for multi-turn benchmarking to avoid confusion.
+
+> [!NOTE]
+> **Dataset Generation vs Request Execution**
+>
+> The `--num-dataset-entries` option controls how many **unique prompts** are generated in the dataset. This is separate from the number of requests or conversations:
+>
+> - `--num-dataset-entries`: Number of unique prompt entries to generate in the dataset
+> - `--request-count`: Number of single-turn requests to send (for single-turn benchmarks)
+> - `--conversation-num`: Number of conversations to send (for multi-turn benchmarks)
+>
+> The dataset entries are reused/sampled as needed to fulfill the total request or conversation count. For example, you might generate 100 unique prompts (`--num-dataset-entries 100`) but send 1000 requests that sample from those prompts.
```

Comment:
You can consider adding that the `--dataset-sampling-strategy` determines how they are reused/sampled.